plugins/wasi_nn/neuralspeed.cpp

```diff
@@ -0,0 +1,338 @@
+#include "neuralspeed.h"
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_NEURAL_SPEED
+#include "simdjson.h"
+#include <time.h>
+#endif
+#include "wasinnenv.h"
+#include <chrono>
+#if !defined(_WIN32) && !defined(_WIN64) && !defined(__WIN32__) &&             \
+    !defined(__TOS_WIN__) && !defined(__WINDOWS__)
+#include <dlfcn.h>
+#endif
+namespace WasmEdge::Host::WASINN::NeuralSpeed {
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_NEURAL_SPEED
+#if defined(_WIN32) || defined(_WIN64) || defined(__WIN32__) ||                \
+    defined(__TOS_WIN__) || defined(__WINDOWS__)
+HINSTANCE SharedLib = LoadLibrary(PYTHON_LIB_PATH);
+#else
+void *SharedLib = dlopen(PYTHON_LIB_PATH, RTLD_GLOBAL | RTLD_NOW);
+#endif
+void printImformation(Graph &GraphRef, Context &CxtRef) {
+  spdlog::info(
+      "[WASI-NN][Info] Neural speed backend: Number of input tokens: {}"sv,
```

Comment:
You don't need to add `[Info]` here, because the `spdlog::info` will do this for you.

Reply:
I have changed. Please check, thank you.